### PR TITLE
catz

### PR DESCRIPTION
**Pull request by chendaniely**
why not have more cats in your code?

randomly...

re submission from #3305

### CODE REVIEW

File: recipes/catzzz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - python
+
+test:
+  imports:
+    - catzzz
+
+about:
+  home: https://github.com/logston/catzzz
+  license: BSD 3-clause
+  license_family: BSD
```
Comment (synapticarbors) [inline recipes/catzzz/meta.yaml]: please include `license_file: LICENSE`

**Comment by ocefpaf** [comment]
Can I haz catzzz?

File: recipes/catzzz/meta.yaml (hunk 1 above)
Comment (chendaniely) [reply recipes/catzzz/meta.yaml]: done.

**Comment by chendaniely** [comment]
@ocefpaf you may have catzzz

**Pull request merged**